agent/config/parse_windows.go

```diff
@@ -32,8 +33,108 @@ const (
 	// domain join check validation. This is useful for integration and
 	// functional-tests but should not be set for any non-test use-case.
 	envSkipDomainJoinCheck = "ZZZ_SKIP_DOMAIN_JOIN_CHECK_NOT_SUPPORTED_IN_PRODUCTION"
+
+	ReleaseId2004SAC        = "2004"
+	ReleaseId1909SAC        = "1909"
+	WindowsServer2019       = "Windows Server 2019"
+	WindowsServer2016       = "Windows Server 2016"
+	WindowsServerDataCenter = "Windows Server Datacenter"
+	InstallationTypeCore    = "Server Core"
+	InstallationTypeFull    = "Server"
+	UnknownOSType           = "unknown"
+	osTypeFormat            = "WINDOWS_SERVER_%s_%s"
+	ecsWinRegistryRootKey   = registry.LOCAL_MACHINE
+	ecsWinRegistryRootPath  = `SOFTWARE\Microsoft\Windows NT\CurrentVersion`
```

Comment:
nit: Should we consider moving these re-usable constants to config/const_windows.go?

Reply:
+1, also please keep the variables unexported if they are not used in other packages.

Reply:
Done. Looks like this package has a separate file for constants which I missed looking at it. However in general  Its a good practice defining constants in the same file as they are being used rather defining some where else.  Thanks on the export , I didn't know the convention in go.